assets/app/scripts/directives/metrics.js

```diff
@@ -0,0 +1,257 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('podMetrics', function($interval, $parse, MetricsService, usageValueFilter) {
+    return {
+      restrict: 'E',
+      scope: {
+        pod: '=',
+        alerts: '='
+      },
+      templateUrl: 'views/directives/_pod-metrics.html',
+      link: function(scope) {
+        var intervalPromise;
+        var getMemoryLimit = $parse('resources.limits.memory');
+        var getCPULimit = $parse('resources.limits.cpu');
+
+        function bytesToMB(value) {
```

Comment:
@spadgett its good to know when a question can be answered with the documentation that I wrote ;-)